apps/frontend/core-dapp/src/features/deposit/DepositLimitWarning.tsx

```diff
@@ -104,13 +102,16 @@ const DepositWarning: React.FC = () => {
       )
     }
     default: {
-      return (
-        <Box>
-          <p>Your ETH will be deposited as wstETH to earn you yield.</p>
-        </Box>
-      )
+      if (showWstEthInfo)
+        return (
+          <Box>
+            <p>Your ETH will be deposited as wstETH to earn you yield.</p>
+          </Box>
+        )
+
+      return null
```

Comment:
Maybe this is cleaner?

```suggestion
        return showWstEthInfo ? (
          <Box>
            <p>Your ETH will be deposited as wstETH to earn you yield.</p>
          </Box>
          ) : null
```